spec/requests/registrations_spec.rb

```diff
@@ -1313,3 +1343,21 @@ def refund_response(capture_id)
     ],
   }.to_json
 end
+
+def payment_confirmation_webhook(intent)
```

Comment:
The fact that you're serializing to JSON should be reflected in the method name.

Alternatively, don't serialize to JSON in this method and serialize the return value after storing it in a variable instead.

Reply:
Changed